scripts/release_binary.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
```

Comment:
I like to include a header with a simple synopsis of what it does and how to use it